source/paysafe.php

```diff
@@ -33,7 +33,9 @@
 
 function __PaysafeAutoloader($className)
 {
-    $classPath = strtolower(str_replace("\\", DIRECTORY_SEPARATOR, $className));
+    // lc first will make the P in the Paysafe root namespace lower case
+    // the remainder of the path/file should match case of the namespace/class
+    $classPath = str_replace("\\", DIRECTORY_SEPARATOR, lcfirst($className));
```

Comment:
Here is the minor autoloader change I mentioned, with comments explaining. This change was inspired by this: https://github.com/paysafegroup/paysafe_sdk_php/issues/11#issuecomment-304372096